platform/security/certificate/security-certificate-generator/src/test/java/org/codice/ddf/security/certificate/generator/CertificateCommandTest.java

```diff
@@ -0,0 +1,228 @@
+        final InputStream systemKeyStream = CertificateGenerator.class.getResourceAsStream(
+                "/serverKeystore.jks");
+
+        IOUtils.copy(systemKeyStream, systemKeyOutStream);
+        IOUtils.closeQuietly(systemKeyOutStream);
+        IOUtils.closeQuietly(systemKeyStream);
+
+        System.setProperty(SecurityConstants.KEYSTORE_TYPE, "jks");
+        System.setProperty(SecurityConstants.KEYSTORE_PATH, systemKeystoreFile.getAbsolutePath());
+        System.setProperty(SecurityConstants.KEYSTORE_PASSWORD, "changeit");
+    }
+
+    @Test
+    public void testConfigureDemoCertWithoutSAN() throws Exception {
+        final KeyStoreFile ksf = CertificateCommand.getKeyStoreFile();
+
+        assertThat(ksf.aliases()
+                .size(), is(2));
+        assertThat(ksf.isKey("my-fqdn"), is(false));
+
+        assertThat(CertificateCommand.configureDemoCert("my-fqdn", null), equalTo("CN=my-fqdn"));
+
+        validateKeyStore("my-fqdn", false);
+    }
+
+    @Test
+    public void testConfigureDemoCertWithSANs() throws Exception {
+        final KeyStoreFile ksf = CertificateCommand.getKeyStoreFile();
+
+        assertThat(ksf.aliases()
+                .size(), is(2));
+        assertThat(ksf.isKey("my-fqdn"), is(false));
+
+        assertThat(CertificateCommand.configureDemoCert("my-fqdn", CertificateCommandTest.SANs),
+                equalTo("CN=my-fqdn"));
+
+        validateKeyStore("my-fqdn", true);
+    }
+
+    @Test
+    public void testConfigureDemoCertWithDNAndWithoutSAN() throws Exception {
+        final KeyStoreFile ksf = CertificateCommand.getKeyStoreFile();
+
+        assertThat(ksf.aliases()
+                .size(), is(2));
+        assertThat(ksf.isKey("CN=John Whorfin,O=Yoyodyne,L=San Narciso,ST=California,C=US"),
+                is(false));
+
+        assertThat(CertificateCommand.configureDemoCertWithDN(new String[] {"cn=John Whorfin",
+                        "o=Yoyodyne", "l=San Narciso", "st=California", "c=US"}, null),
+                equalTo("CN=John Whorfin,O=Yoyodyne,L=San Narciso,ST=California,C=US"));
+
+        validateKeyStore("john whorfin", false);
+    }
+
+    @Test
+    public void testConfigureDemoCertWithDNAndSANs() throws Exception {
+        final KeyStoreFile ksf = CertificateCommand.getKeyStoreFile();
+
+        assertThat(ksf.aliases()
+                .size(), is(2));
+        assertThat(ksf.isKey("CN=John Whorfin,O=Yoyodyne,L=San Narciso,ST=California,C=US"),
+                is(false));
+
+        assertThat(CertificateCommand.configureDemoCertWithDN(new String[] {"cn=John Whorfin",
+                        "o=Yoyodyne", "l=San Narciso", "st=California", "c=US"},
+                CertificateCommandTest.SANs),
+                equalTo("CN=John Whorfin,O=Yoyodyne,L=San Narciso,ST=California,C=US"));
+
+        validateKeyStore("john whorfin", true);
+    }
+
+    @Test
+    public void testMainWithCNAndWithoutSAN() throws Exception {
```

Comment:
I had to add them to increase coverage. There are different paths in the main() method with these different arguments.